omod/src/main/resources/webModuleApplicationContext.xml

```diff
@@ -129,11 +139,39 @@
 				
 				<prop key="admin/provider/providerAttributeType.list">providerAttributeTypeList</prop>
 				<prop key="admin/provider/providerAttributeType.form">providerAttributeTypeForm</prop>	
-				
+
  			</props>
 		</property>
 	</bean>	
 
+ 	<!-- index pages - begin -->  
+	
+	<!-- admin -->
+	<mvc:view-controller view-name="module/legacyui/admin/index" path="admin/index.htm"/>				
```

Comment:
@jdegraft maybe there is something that you are aware of that is special about /admin. I'm not quite sure why your initial suggestion of just mapping /admin wouldn't work, as in
<mvc:view-controller view-name="module/legacyui/admin/index" path="/admin"/>. If that doesn't, 
I think it's at least worth trying the controller with @RequestMapping(value="/admin")